docs/00-manuscripts/ieee/ieee-manuscript.tex

```diff
@@ -1217,7 +1217,16 @@ \section{Vegetation index formulas}
 
 \section{Radiometric, Geometric and Atmospheric correction of hyperspectral data}
 
-The following information was provided externally by the company which was in charge for the image acquisition and preprocessing of the data.
+The following information was provided by the Institut Carogràfic i Geològic de Catalunya, which was in charge of image acquisition and data preprocessing.
+
+
+\textbf{Sensor Characteristics}
+
+\begin{quote}
+The AisaEAGLE II sensor was used for airborne image acquisition with a field of view of 37.7°.
+Its spectral resolution is 2.4 nm in the range from 400 to 1000 nm.
+\end{quote}
```

Comment:
I would move this part into section "Hyperspectral data" after merging.

What is the motivation to quote this part?